SEP-0002-revised.md

```diff
@@ -0,0 +1,245 @@
+The advisory board shall advise and consent on those plans.
+
+## The Steering Committee
+The role of the steering committee which is headed by the Lead Developer, is to manage the day to day operations of the SunPy organization.
+The steering committee shall be responsible for the running of the project under advisement of the board.
+This includes:
+
+* holding regular meetings with the developer community (at least monthly)
+* making regular reports to the board (at least quarterly)
+* having ownership privileges to the SunPy GitHub organization
+* creating, maintaining and enforcing the SunPy code of conduct (see below)
+* accepting or rejecting SEPs based on community consensus
+* delegating responsibility and authority to other community roles
+* planing and manage finances and financial processes of the project.
+
+### Composition
+
+The steering committee is formed of three roles:
+
+* **The lead developer** chairs the steering committee and is responsible for holding regular (at least monthly) meetings, and ensuring that the committee is represented at all board meetings.
+* **The deputy lead developer** in the absence of the lead developer, the deputy shall perform all the duties of the lead developer.
+* **The financial officer** the financial officer is primarily responsible for ensuring the health of the project finances and setting up processes for managing expenses and reporting to the board on financial matters.
+
+The steering committee is elected by the **project members**.
+
+### Decision Making
+
+The steering committee should strive to make decisions by consensus.
+If consensus can not be reached decisions can be taken by a majority vote.
+
+### Electing the Steering Committee
+
+An election to fill a vacant role on the steering committee will occur in two stages:
+
+* Nominations: No less than three weeks before an election a public announcement of the election must be made and nominations solicited for no less than two weeks.
+Candidates which have accepted their nomination (self nominations are allowed) should write a short statement which will be made public as soon as possible, but no later than before the election starts.
+* Election: All candidates that have written a short statement will be put to a vote of the project members and the candidate with the most votes shall be elected to the position.
+In the event that more than one position has to be filled simultaneously each election shall be treated as a separate election for the purposes of this process.
+
+### Term
+
+Each position on the steering committee has a term of three years, from the date of the announcement of the election results.
+
+If there is a vacancy part way through a term, then a by-election will be held where the candidate elected to the committee will serve the remainder of the term of the member they are replacing.
```

Comment:
a by-election "can be held" instead of "will be"? More flexibility since we probably don't want to run elections all the time.